26.md

```diff
@@ -52,6 +52,8 @@ For example, the following condition strings are valid:
 - `kind=0&kind=1&created_at>1675721813`
 - `kind=1&created_at>1674777689&created_at<1675721813`
 
+For the vast majority of use-cases, it is advisable that query strings should include a `created_at` ***after*** condition reflecting the current time, to prevent delegatee's from publishing historic notes on delegator's behalves.
```

Comment:
Why "delegatee's"? Is this a typo, or am I parsing it wrong?

```suggestion
For the vast majority of use-cases, it is advisable that query strings should include a `created_at` ***after*** condition reflecting the current time, to prevent the delegatee from publishing historic notes on delegator's behalves.
```

Reply:
It's not a typo, it's advice for the plural 'query strings' (of different tokens), which would have different delegators and delegatees. Your suggestion is less ambiguous so I'm happy to go with that, however I think that there has to be a change in the last three words also:

> For the vast majority of use-cases, it is advisable that query strings should include a `created_at` ***after*** condition reflecting the current time, to prevent the delegatee from publishing historic notes on the delegator's behalf.

Reply:
I still struggle to parse "to prevent delegatee's from publishing"… Why the apostrophe? "delegatees" (plural) makes sense here. It's delegatees that are prevented from publishing historic notes. I don't get the apostrophe in this sentence.

Reply:
Ah, understood, that is indeed a typo then - the reworded suggestion threw me off. Would you like to suggest your now preferred change and I can add it?

Reply:
Sorry for the confusion, I wasn't sure whether the error was in the pluralization or something else.

Either version is ok for me, use your own personal preference.

Reply:
I can't resolve the conversation because I didn't open the PR and I don't have write access. Feel free to mark as resolved, however.

Reply:
Ah, I've been using Gitlab for the last few years so I'm no longer as familiar with Githubs PR process. Thank you very much, I'll resolve now.